minkyu/app.js

```diff
@@ -36,10 +39,40 @@ app.use(express.json());
 app.use(cors());
 app.use(morgan('tiny'));
 
-app.get("/ping", (req,res) =>{
-    res.json({message : "pong"});
+app.get("/pings", (req,res) =>{
+    res.status(201).json({"message" : "pong"});
 });
 
+app.post("/users", async(req, res) =>{
+    const { name, email, password } = req.body;
+
+    await database.query(
+        `INSERT INTO users(
+            name,
+            email,
+            password
+        ) VALUES (?, ?, ?, ?);
+        `,
+        [ name, email, password ]
+    );
+
+    res.status(201).json({ message : "successfully created" });
+})
+
+app.post("/posting", async(req, res) => {
+    const {title, content, user_id} = req.body;
```

Comment:
```suggestion
    const {title, content, userId} = req.body;
```
javascript 에서는 snake_case가 아닌 camelCase를 사용하도록 해주세요-!